src/components/Textarea.tsx

```diff
@@ -1,27 +1,37 @@
+import {
+	TextArea as GrommetTextArea,
+	TextAreaProps as GrommetTextAreaProps,
+} from 'grommet';
 import * as React from 'react';
 import styled from 'styled-components';
 import asRendition from '../asRendition';
 import { DefaultProps, RenditionSystemProps } from '../common-types';
-import { radius } from '../theme';
-import { monospace, px } from '../utils';
+import { monospace } from '../utils';
+
+const getBorderColor = (props: InternalTextareaProps & { theme: any }) => {
+	if (props.invalid) {
+		return props.theme.colors.danger.main;
+	}
+	if (props.valid) {
+		return props.theme.colors.success.main;
+	}
+};
+
+const StyledGrommetTextArea = styled(GrommetTextArea)`
+	${monospace};
+
+	padding: 15px 20px;
```

Comment:
Let's have this as one of the spacing array from the theme (I guess 16px both axes)

Reply:
Ah, ok. Just went with the designs.

Reply:
@dimitrisnl actually I also see that for the Input and Select component we use `14px 20px`, so maybe go with that for consistency.